apps/web/vercel.json

```diff
@@ -30,5 +30,8 @@
     "pages/api/book/event.ts": {
       "memory": 2048
     }
+  },
+  "git": {
+    "cloneDepth": 1000
```

Comment:
To avoid error: Could not find the SHA of the previous release in the git history.

Reply:
Vercel can handle it: 

> During this time, Vercel performs a "shallow clone" on your Git repository using the command git clone --depth=10 (...) and fetches ten levels of git commit history. This means that only the latest ten commits are pulled and not the entire repository history.

https://vercel.com/docs/builds/configure-a-build

Reply:
IMO, we need to change the build command as mentioned before, so Sentry cli can handle it automatically without any changes.

Reply:
Will try again without this but Setting this got rid of the error.

Reply:
We cannot change the build command as we need to build various dependencies, hence the update.

Reply:
Sure

Reply:
> We cannot change the build command as we need to build various dependencies, hence the update.

What if we just use the simple build command above, does the app work？

Reply:
We need to build embeds. icons and other dependencies. The build command alone bypasses turbo-repo dependency chain.

Reply:
Can we have a test👀

Reply:
The app works with the simple build command. Already tested it. But we cannot merge it that way. We need to keep the original build command.

Reply:
If we ignore the git commits, it’s not easy to find which commit introduced the bug.